state_machine/drivers/pycubedmini/lib/logging.py

```diff
@@ -0,0 +1,187 @@
+from os import listdir, stat, statvfs, remove, mkdir, rmdir
+import time
+import lib.pycubed as cubesat
+
+info_filename = "info.txt"
+info_filename_dir = "/sd/info.txt"
```

Comment:
I'd try to avoid duplicating the contents of `info_filename`. Also, let's name it `loginfo.txt`.
```
info_filename = "loginfo.txt"
info_file_dir = "/sd/"
info_filename_dir = info_file_dir + info_filename
```

Reply:
Also, a standard convention is for constant names to be in CAPS. I'm thinking we are generally following this, but not positve. @thetazero can you confirm if this is a convention we're holding to?